src/Sentry/Scope.cs

```diff
@@ -461,6 +464,8 @@ public void Apply(Scope other)
     public Scope Clone()
     {
         var clone = new Scope(Options);
+        clone.OnEvaluating = OnEvaluating;
```

Comment:
This was required to ensure the event handler attached in the middleware is still called.  Otherwise, the new scope has no handlers attached to this event.